doc/api/dgram.md

```diff
@@ -239,8 +239,9 @@ respect to [byte length][] and not the character position.
 If `msg` is an array, `offset` and `length` must not be specified.
 
 The `address` argument is a string. If the value of `address` is a host name,
-DNS will be used to resolve the address of the host. If the `address` is not
-specified or is an empty string, `'127.0.0.1'` or `'::1'` will be used instead.
+DNS will be used to resolve the address of the host. Otherwise, if `address` is
+not specified or is an empty string, `'127.0.0.1'` (`udp4`) or `'::1'` (`udp6`)
```

Comment:
I think he is saying that is current behaviour, and not doing it would be semver-major. I would say if we are going to change the API, I would expect only allow `null` and `undefined` to mean "no value". Btw, from above, it looks like `42` would also be considered a host name? I.e., numbers are allowed? Do they get converted to strings, like `'42'` and passed to the resolver?